kwanghyun/app.js

```diff
@@ -50,6 +50,54 @@ app.post("/users", async (req, res) => {
   );
   res.status(201).json({ message: "userCreated" });
 });
+app.post("/likes", async (req, res) => {
+  const { user_id, post_id } = req.body;
+  await appDataSource.query(
+    `INSERT INTO likes(
+        user_id,
+        post_id
+      ) VALUES (?, ?)
+      `,
+    [user_id, post_id]
+  );
+  res.status(201).json({ message: "likeCreated" });
+});
+
+app.patch("/user/post/modify", async (req, res) => {
```

Comment:
- endpoint에는 명사만 사용해 주세요.
- 이미 HTTP method의 patch로 어떠한 행위를 할지 정해졌기 때문에 endpoint에서 patch는 제거해주셔도 됩니다.
```suggestion
app.patch("/user/post", async (req, res) => {
```